### PR TITLE
[trambyt] Add trambyt app

### PR DESCRIPTION
**Pull request by protocol7**
Shows upcoming departures from Västtrafik, western Sweden public transport. The use can configure a transit stop and the app will show the next departures.

### CODE REVIEW

**Comment by protocol7** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/trambyt/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Trambyt
+
+Shows upcoming departures from Västtrafik, western Sweden public transport. The use can configure
+a transit stop and the app will show the next departures.
+
+![Trambyt screengrab](trambyt.png)
+
+Dedicated to matslina for his deep love of fish.
```
Comment (matslina) [inline apps/trambyt/README.md:8]: I hope you're running [Fishbyt](https://github.com/tidbyt/community/blob/main/apps/fishbyt/fishbyt.star) on your Tidbyt!

**Review by matslina — COMMENTED** [review]
=D

File: apps/trambyt/trambyt.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+var source []byte
+
+// New creates a new instance of the Trambyt applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "trambyt",
+		Name:        "Trambyt",
+		Author:      "protocol7",
+		Summary:     "Departures for Västtrafik",
+		Desc:        "Show departures for Västtrafik stops.",
```
[the remark this answers is not in the document]
Comment (protocol7) [reply apps/trambyt/trambyt.go:20]: Made an attempt. I could figure out if I could test this flow at all as pixlet doesn't seem to support LocationBased, so this is a bit of a gamble :)

**Review by betterengineering — APPROVED** [review]

**Pull request merged**